grails-app/services/org/pih/warehouse/inventory/CycleCountProductAvailabilityService.groovy

```diff
@@ -124,7 +124,7 @@ class CycleCountProductAvailabilityService {
                     it.inventoryItem == cycleCountItem.inventoryItem &&
                     it.custom
         }
-        if (!wasItemEverCustomAdded) {
+        if (!wasItemEverCustomAdded && cycleCountItem.quantityCounted <= 0) {
```

Comment:
Why <= 0 and not == 0? We can't count a negative number right? I suppose it's fine this way though.

Reply:
yeah, just to be extra sure I made it `<= 0`. Feels like we don't have any `min/max` validation neither in command nor in the domain on the `quantityCounted` field so potentially a user might send a negative value through the API.